functions/source/faz-handler/lib/core/autoscale-handler.js

```diff
@@ -656,7 +656,7 @@ module.exports = class AutoscaleHandler {
                 // 1. a new instance comes up and becomes the new master, master healthcheck won't
                 // exist yet because this instance isn't added to monitor.
```

Comment:
```suggestion
                // exist yet because this instance hasn't been added to be monitored.
```

Reply:
I didn't take it change. minor and irrelevant.